docs/README.md

```diff
@@ -2,13 +2,13 @@
 
 A built version of document is available at http://mxnet.io
 
-To build the documents locally, we need to first install [docker](docker.com).
+To build the documents locally, we need to first install [docker](https://docker.com).
 Then use the following commands to clone and
 build the documents.
 
 ```bash
 git clone --recursive https://github.com/apache/incubator-mxnet.git
-cd mxnet && make docs
+cd incubator-mxnet && make docs
```

Comment:
This seems to be happening at a lot of places after we moved repo. Should we change to 
`git clone --recursive https://github.com/apache/incubator-mxnet.git mxnet` everywhere? @kevinthesun

Reply:
@piiswrong excellent suggestion! 

Let me know if you've like me to remove this particular change so you can defer it to a larger discussion/pr.

Reply:
This is all the places where this is needed.
```
c4b301d74d83:mxnet junyuanx$ git grep "https://github.com/apache/incubator-mxnet.git"
docs/README.md:git clone --recursive https://github.com/apache/incubator-mxnet.git
docs/build_version_doc/AddVersion.py:                                        'git clone --recursive https://github.com/apache/incubator-mxnet.git')
docs/build_version_doc/AddVersion.py:                                        'git clone --recursive https://github.com/apache/incubator-mxnet.git '
docs/build_version_doc/build_all_version.sh:mxnet_url="https://github.com/apache/incubator-mxnet.git"
```
Would you like to do the change at these 3 places?

Reply:
Yeah I'd be happy to